api.swagger.yaml

```
@@ -9,6 +9,23 @@ security:
   - APISessionHeader: []
   - APIKeyHeader: []
 paths:
+  /status:
+    get:
+      tags:
+        - status
+      responses:
+        '200':
+          description: Status returned successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    description: Status of the API
+                    type: string
+                    example: 'Ok'
+      description: Returns the status of the Invotra API. If successful, the API is running as expected.
```

```suggestion
      description: Returns the status of the Invotra API. If successful, the API is running as expected. (If it is not, an appropriate 5xx error message will be returned).
```

While we don't explicitly document 5xx errors, I think it's worth adding an explanation of why this endpoint doesn't have a negative response.